app/views/spree/admin/orders/_print.pdf.prawn

```diff
@@ -4,7 +4,7 @@ require 'prawn/layout'
 
 font @font_face
 
-im = Rails.application.assets.find_asset(Spree::PrintInvoice::Config[:print_invoice_logo_path])
+im = Sprockets::Railtie.build_environment(Rails.application).find_asset(Spree::PrintInvoice::Config[:print_invoice_logo_path])
```

Comment:
Is there a reason you can't use `asset_path` for this? Just asking, I'm not familiar with this API.

Reply:
Hi @aldesantis . `Rails.application.assets` is nil in production mode as the assets are served by `sprockets` and are compiled already. For the `prawn` templates we need the actual file object to get the path locally and we might need some context of the `Rails.root` and how the app folders are in the project. In this way we could get the asset file.

Reply:
@memotoro ah, sorry, I didn't realize how it was being used in that template.